src/infra/main/template.yaml

```diff
@@ -1346,6 +1399,146 @@ Resources:
       AliasName: !Sub "alias/${AWS::StackName}-CloudWatchEncryptionKey"
       TargetKeyId: !GetAtt CloudWatchEncryptionKey.Arn
 
+  #
+  #  Canary Deployment CloudWatch Alarms
+  #
+
+  StatusRetrieverFunctionErrorsAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Ref AlertingSNSHighAlertNotificationTopicARN
+      AlarmDescription: "Canary Deployment: Errors returned from the StatusRetrieverFunction Lambda"
```

Comment:
Could we remove `Canary Deployment: ` prefix as these alarm could still be triggered outside of the cotext of deployments